crates/sui-graphql-rpc/src/extensions/logger.rs

```diff
@@ -25,7 +25,7 @@ pub struct LoggerConfig {
 impl Default for LoggerConfig {
     fn default() -> Self {
         Self {
-            log_request_query: true,
+            log_request_query: false,
             log_response: true,
```

Comment:
Probably also turn this off by default? (The response will generally be larger than the query).